examples/README.md

```diff
@@ -0,0 +1,15 @@
+# How to use Modular Adversarial Robustness Toolkit
+
+We provide examples on how to use the toolkit in your project.
+
+A typical procedure is
+
+1. Install the toolkit as a Python package `pip install https://github.com/IntelLabs/MART/archive/refs/heads/main.zip`;
```

Comment:
Probably should tag a version to install. Otherwise main will continually change.

Reply:
I added instructions on installing a released version. 

This is a general README for all examples and they may depend on different versions in the future.